src/RA_Defs.h

```diff
@@ -11,6 +11,8 @@
 #include <queue>
 #include <deque>
 #include <map>
+#include <atlbase.h>
```

Comment:
why was this necessary?

Reply:
It's not. At least it wasn't here. It is needed for CComPtr though. I think I was going to that first but thought I should do something simpler.

Reply:
This is gone